deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -0,0 +1,51 @@
+receivers:
+  opencensus: {}
+  jaeger:
+    protocols:
+      grpc:
+  zipkin: {}
+processors:
+  k8s_tagger:
+    passthrough: false
+    extract:
+      metadata:
+        # extract the following well-known metadata fields
+        - podName
+        - deployment
+        - cluster
+        - namespace
+        - node
+        - startTime
+  queued_retry:
+    num_workers: 16
+    queue_size: 10000
+    retry_on_failure: true
+  batch:
+    send_batch_size: 128
```

Comment:
I think the default was 1024 which I decreased to make sure we are not waiting too long for packages, but I think we might up it to at least 512 (if not the OT default)

Reply:
👍